test/y2network/sysconfig/config_writer_test.rb

```diff
@@ -137,8 +137,9 @@
       let(:old_config) do
         instance_double(
           Y2Network::Config,
-          dns:        double("dns"),
-          interfaces: [eth0, eth1]
+          dns:         double("dns"),
+          interfaces:  [eth0, eth1],
```

Comment:
Actually, interfaces should be an `InterfacesCollection` object and connections an `ConnectionConfigsCollection`